user/enterprise/install-on-xenial.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Install on Ubuntu 16.04 machines (Beta)
+title: Install onto Ubuntu 16.04 machines (Beta)
```

Comment:
should we say install Travis CI Enterprise on Ubuntu... ?

I'm kinda in favor of install on vs onto as well. But :man_shrugging:

Reply:
yeah, I can switch back 👍